src/constants/base-contracts.ts

```diff
@@ -0,0 +1,96 @@
+const baseContracts = {
+  ERC721Base: `// SPDX-License-identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@thirdweb-dev/contracts/base/ERC721Base.sol";
+
+contract MyNFT is ERC721Base {
+
+      constructor(
+        string memory _name,
+        string memory _symbol,
+        address _royaltyRecipient,
+        uint128 _royaltyBps
+    )
+        ERC721Base(
+            _name,
+            _symbol,
+            _royaltyRecipient,
+            _royaltyBps
+        )
+    {}
+
+}`,
+  ERC721Signature: `// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@thirdweb-dev/contracts/base/ERC721SignatureMint.sol";

```

Comment:
All of these also follow a very similar format - seems like the only thing that's really different is the constructor params of each base contract + the import name. Might be an easy way to fill these in instead of having a separate string for each. But we don't need to worry about this for now.